src/front/glsl/constants.rs

```diff
@@ -506,6 +557,48 @@ impl<'a> ConstantSolver<'a> {
     }
 }
 
+/// Helper function to implement the glsl `max` function for floats
+///
+/// While rust does provide a `f64::max` method it has a different behavior than the
+/// glsl `max` for `NaN`s, in rust if any of the arguments is a `NaN` then the other
+/// is returned.
```

Comment:
Capitalize "Rust", and write "GLSL" in all caps. Also, this is a run-on sentence; please change ", in rust if" to ". In Rust, if"